### ISSUE TITLE
what is the exact validation set for fishyscapes lost and found?

### ISSUE THREAD

**Issue by suesuekkim** [hermannsblum/fishyscapes#12]
Hello, I'm trying to utilize some stuff with fishyscapes dataset about anomaly segmentation task.

I have followed the instruction in the fishyscapes dataset official site (https://fishyscapes.com/dataset),
and i succeed to download the dataset. 

I donwloaded 'fishyscapes_lostandfound', that you uploaded on upper site which was mentioned in the button 'download FS Lost&Found validation set' and it was included of 100 masked images.
Also importing bdlb, I could download gtCoarse and leftImg8bit and it contained train folder and test fold.

So to implement training, validation, and test with these datas, do I need to split validation set inside the training set of gtCoarse and leftImg8bit? Then what is the 'fishyscapes_lostandfound' validation data? 

Also I could not get the fishyscapes static dataset with your official site. There was nothing in the folder 'static'.

And is there any gtFine dataset in fishyscapes? 

Thank you for reading my long issue, and I really appreciate your work of Fishyscapes benchmark.

**Comment by hermannsblum** [comment]
Hi, I will try to answer all points, but please let me know if you have further questions!

- The data should not be used for training. To train your model, you should use the cityscapes dataset. The cityscapes dataset also has gtFine.
- You can use the fishyscapes_lostandfound for validation, as we also recommend in the benchmark paper.
- We do not release the data of the test split. To test your model on the testset, you have to submit your code at https://fishyscapes.com/submission
- I am aware of the issues with the FSStatic validation split (which is also only 20 images for visual inspection, not really useful for anything else). I will try to fix the issue, but cannot give you a timeline.

**Comment by hermannsblum** [comment]
@suesuekkim as an update, the issues with the fishyscapes static dataset, you should now be able to download it after preparing the Cityscapes data as explained here: https://www.tensorflow.org/datasets/catalog/cityscapes
